tools/lint-docstring

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import importlib
+import pathlib
+import sys
+from os.path import basename, dirname, join, relpath
+from typing import Dict, Tuple, cast
+
+
+INITIAL_INDEX = 7
+
+# absolute path to zulip-terminal
+ROOT_DIRECTORY = pathlib.Path(__file__).resolve().parent.parent
+
+# absolute path to zulip-terminal/zulipterminal to be passed as parameter
+ZULIPTERMINAL = join(ROOT_DIRECTORY, "zulipterminal")
+
+# absolute path to the folders under zulipterminal/ to be skipped while parsing
+THEMES_PATH = join(ZULIPTERMINAL, "themes")
+SCRIPTS_PATH = join(ZULIPTERMINAL, "scripts")
```

Comment:
This does work, but is fragile - if we change this behavior or add other folders like this we need to do extra work, and we're already doing it twice. It is cleaner to make a data-structure that represents special cases and then process it. That will allow it to include the 4 other constants you defined lower down.